test/test_manifest.json

```
@@ -7292,5 +7292,12 @@
       "rounds": 1,
       "link": true,
       "type": "eq"
+   },
+   {
+      "id": "issue15289",
+      "file": "pdfs/issue15289.pdf",
+      "md5": "f3d20aee398e052ac3dce7cc030f50de",
+      "rounds": 1,
+      "type": "eq"
```

It should suffice to test only one page, right?
```suggestion
   {
      "id": "issue15289",
      "file": "pdfs/issue15289.pdf",
      "md5": "f3d20aee398e052ac3dce7cc030f50de",
      "rounds": 1,
      "lastPage": 1,
      "type": "eq"
```